tiatoolbox/dataloader/wsireader.py

```diff
@@ -137,5 +145,170 @@ def read_region(self, start_w, start_h, end_w, end_h, level=0):
         im_region = openslide_obj.read_region(
             [start_w, start_h], level, [end_w - start_w, end_h - start_h]
         )
-        im_region = np.asarray(im_region)
+        im_region = background_composite(image=im_region)
         return im_region
+
+    def slide_thumbnail(self):
+        """Read whole slide image thumbnail at 1.5x
+
+        Args:
+            self (WSIReader):
+
+        Returns:
+            ndarray : image array
+
+        Examples:
+            >>> from tiatoolbox.dataloader import wsireader
+            >>> wsi_obj = wsireader.WSIReader(input_dir="./",
+            ...     file_name="CMU-1.ndpi")
+            >>> slide_thumbnail = wsi_obj.slide_thumbnail()
+
+        """
+        openslide_obj = self.openslide_obj
+        tile_objective_value = 20
```

Comment:
what is this? im getting mixed up here between objective power and objective power - please add a comment in code to explain

Reply:
objective power is the magnification at level 0. tile_objective_value is magnification at which the tile is read. Updated in this commit https://github.com/TIA-Lab/tiatoolbox/commit/6b129f7936a9f10400d97d8e623c3cbdc04dc02d